pages/questionnaireCrowd.js

```diff
@@ -273,17 +356,20 @@ class QuestionnaireCrowdPage extends Component {
   }
 
   submit(errorMessages) {
+    const { addMeToRole } = this.props
+    const { willingnessStatus } = this.state
     const hasErrors = this.processErrors(errorMessages)
     if (hasErrors) return
 
     this.setState({ updating: true })
     this.processSubmit()
-      .then(() =>
-        this.setState(() => ({
+      .then(() => {
+        willingnessStatus === 'true' && addMeToRole({ role: 'accomplice' })
+        return this.setState(() => ({
```

Comment:
We should add `addMeToRole` to the promise chain: ensuring the error is handled.
```suggestion
      .then(() => {
        if (willingnessStatus === 'true') {
          return addMeToRole({ role: 'accomplice' })
        }
      })
      .then(() => {
        return this.setState(() => ({
```

Reply:
also after `addMeToRole` we should also auto subscribe to the NL in a additional `then`.

Reply:
Shoud we? I spoke with Carla on Thursday and we agreed to the show the subscription checkbox in the thank you screen and let the people decide if they want to subscribe or not.

Reply:
My opinion ist yes: if they don't even want a newsletter about helping us initially, why even bother helping us? But practically it should be linked to the question «Wie dürfen wir Sie kontaktieren? Wählen Sie so viele Optionen, wie Sie wollen.». I didn't realize that they still have that one and thereby that they make it optional to be contacted by email. This is IMO a mistake on their part.